pipelines/tekton/aiedge-e2e/aiedge-e2e.pipelinerun.yaml

```diff
@@ -29,7 +29,7 @@ spec:
   - name: git-revision
     value: "main"
   - name: test-endpoint
-    value: "invocations"
```

Comment:
Was this change intentional since the `invocations` endpoint was used for the `bike-rentals-auto-ml` model?

Reply:
Whoops, must've made a mistake while rebasing :sweat_smile:. Going to revert this.